amida-tech/indaba-client
# Bug/inba 893 jf activate fix

**Pairs with the backend change PR located [here.](https://github.com/amida-tech/greyscale/pull/491)**

#### What does this PR do?
Prevents users from entering bad passwords and no first name into their activation fields, and gives better instructions and alerts when they try. 

#### Related JIRA tickets:

#### How should this be manually tested?
Login as admin and invite yourself through your Amida account.

You should receive an email that directs you to a page on Indaba. There you can test:
1. No password provided. **(Reject - front end, no POST)**
2. Invalid password. (Reject)
3. Auth service is off. (Reject)
4. Greyscale is off. (Reject)
5. Valid password but no first name. **(Reject - front end, no POST)**
6. Valid password and first name. (Success, redirect to sign in, and sign in is successful)

#### Background/Context

#### Screenshots (if appropriate):
